jaxb-ri/xjc/pom.xml

```diff
@@ -145,7 +145,7 @@
                         <configuration>
                             <outputDirectory>${project.build.directory}/xjctask-cp</outputDirectory>
                             <includeArtifactIds>
-                                jakarta.activation,istack-commons-tools,istack-commons-runtime,
+                                jakarta.activation-api,istack-commons-tools,istack-commons-runtime,
```

Comment:
angus-activation should probably be added here

Reply:
Added.